test/files.spec.js

```diff
@@ -62,6 +62,21 @@ describe('.files (the MFS API part)', function () {
       })
     })
 
+    it('file.add with NPM Buffer', (done) => {
+      let Buffer = require('buffer/').Buffer
```

Comment:
Why the `/`?

Reply:
It's to depend on that module specifically. I tested without it and it was using Nodejs core Buffer module.
I read about the [usage here](https://www.npmjs.com/package/buffer).
> To require this module explicitly, use require('buffer/') which tells the node.js module lookup algorithm (also used by browserify) to use the npm module named buffer instead of the node.js core module named buffer!

Reply:
There is no `buffer` package [being imported](https://github.com/joaosantos15/js-ipfs-api/blob/master/package.json). Am I missing some point?